newIDE/app/src/UI/QrCode.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import * as React from 'react';
+import QrCreator from 'qr-creator';
+import GDevelopThemeContext from './Theme/GDevelopThemeContext';
+
+type Props = {|
+  url: string,
+  size?: number,
+|};
+
+const QrCode = ({ url, size = 128 }: Props) => {
+  const gdevelopTheme = React.useContext(GDevelopThemeContext);
+
+  const containerRef = React.useRef<?HTMLDivElement>(null);
+  React.useEffect(
+    () => {
+      const containerElement = containerRef.current;
+
+      if (!containerElement) return;
+
+      QrCreator.render(
```

Comment:
that's a weird way of rendering itself... too bad we can't just call it where we want it to be displayed

Reply:
There was some react libraries but they were a bit too heavy/overkill